src/NuGet.Core/NuGet.Protocol/HttpSource/HttpRetryHandler.cs

```diff
@@ -96,10 +122,21 @@ public class HttpRetryHandler : IHttpRetryHandler
                         if (response.Content != null)
                         {
                             var networkStream = await response.Content.ReadAsStreamAsync();
-                            var newContent = new DownloadTimeoutStreamContent(
+                            var timeoutStream = new DownloadTimeoutStream(requestUri, networkStream, request.DownloadTimeout);
+                            var diagnosticsInfo = new ProtocolDiagnosticEvent(DateTime.MinValue,
```

Comment:
Move 1st argument to next line.

Why are you passing `DateTime.MinValue` and `TimeSpan.MinValue`?  How about another constuctor with those parameters removed?